src/Config.php

```diff
@@ -198,6 +198,23 @@ class Config
      */
     private static $executablePaths = [];
 
+    /**
+     * A list of valid generators.
+     *
+     * - Keys: lowercase version of the generator name.
+     * - Values: name of the generator PHP class.
```

Comment:
Should this be part of the "long description" of the variable or part of the description of the `@var` type ?

Reply:
Makes sense, thanks for the suggestion.